avocado_vt/plugins/vt_cluster.py

```
@@ -0,0 +1,249 @@
+        both the nodes and the cluster manager. If any setup step fails, it
+        logs the error and terminates the job.
+
+        :param job: The Avocado job object.
+        :type job: avocado.core.job.Job
+        """
+        if cluster.get_all_nodes():
+            try:
+                self._setup_nodes()
+                self._setup_mgr()
+                node_properties.save_properties()
+            except Exception as detail:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)